util/mock/context.go

```diff
@@ -63,16 +64,16 @@ func (c *Context) GetGlobalStatusVar(ctx context.Context, name string) (string,
 	if v == nil {
 		return "", terror.UnknownStatusVar.Gen("unknown status variable: %s", name)
 	}
-	return v.Value, nil
+	return types.ToString(v.Value)
 }
 
 // SetGlobalStatusVar implements GlobalVarAccessor SetGlobalStatusVar interface.
-func (c *Context) SetGlobalStatusVar(ctx context.Context, name string, value string) error {
+func (c *Context) SetGlobalStatusVar(ctx context.Context, name, value string) error {
 	v := variable.GetStatusVar(name)
 	if v == nil {
 		return terror.UnknownStatusVar.Gen("unknown status variable: %s", name)
 	}
-	v.Value = value
+	v = &variable.StatusVal{Scope: variable.DefaultScopeFlag, Value: value}
```

Comment:
Seems this line is not work, for the map value is a pointer, if assign it to a new struct address, then next time may not get new value from `variable.GetStatusVar(name)` i think.